go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/etcd-io/auger
 
-go 1.22.4
+go 1.22.6
```

Comment:
https://github.com/etcd-io/auger/blob/5bc9a4f4164e21aabed3605b9382092bb7a0e4df/Dockerfile#L15

the Dockerfile also needs to be changed, the [kubernetes-sigs/zeitgeist](https://github.com/kubernetes-sigs/zeitgeist) can check whether to keep the same in CI, so maybe we can introduce it for that.

Reply:
Thanks - updated to pin dependency to https://hub.docker.com/layers/library/golang/1.22.6-alpine/images/sha256-58e52f6ddd39098d23b1a34ec24024acd5fe182245960afe572d83c313aafbed?context=explore and added dependabot config for auto bump.